### ISSUE TITLE
Alt text missing on location images - UG/PG homepages

### ISSUE THREAD

**Issue by jt353-unikent** [unikent/of-course#503]
Add alt text to location images.

**Comment by eg270-unikent** [comment]
Looks like alt text has been applied to the PG locations but not to UG
